q2-structure/proposal-by-xdg.md

```diff
@@ -0,0 +1,278 @@
+# Proposal for Q2: Structure of Governance
+
+## Definitions
+
+This document describes three roles involved in governance:
+
+* "Core Team"
+
+* "Steering Council"
+
+* "Vote Administrator"
+
+A section on each follows.
+
+The term "Perl Rules of Governance" means the final governance document to be
+written and ratified by the Gov-Begin group.
+
+## The Core Team
+
+The Core Team are a group of trusted volunteers involved in the ongoing
+development of the Perl language and interpreter.  They are not required to be
+language developers or committers.
+
+References to specific votes are explained in the "Rules for Voting" section.
+
+### Powers
+
+The Core Team has authority over the rules and composition of Perl governance.
+
+They alone elect the Steering Council and have the power to remove Steering
+Council members.
+
+In concert with the Steering Council, they manage their own membership and
+have the power to modify the Perl Rules of Governance.
+
+The Core Team do not have any authority over parts of the Perl ecosystem
```

Comment:
Similar to the Council, I'm not sure why we would want to specify what this group cannot do. If you simply say project, then whatever the "project" becomes is in scope.

Reply:
This was already agreed in the Q1 vote so not open to change here.  It's here to make the rest of the section read better as we apply the Q1 rules to the two governance bodies.

Reply:
This is why piecemealing this is problematic.